userguide/tutorials/invoice_examples.adoc

```diff
@@ -0,0 +1,268 @@
+----
+
+[start=3]
+. Finally, when the invoice is committed, the balance is updated:
+
+[source,java]
+----
+INVOICE BALANCE = CHARGED AMOUNT + SUM(CBA_ADJ) - (AMOUNT PAID + AMOUNT REFUNDED)  
+                = 80 + 0 - (0 + 0)  
+                = 80
+----
+
+
+== Invoice Item Adjustment
+
+It is possible to https://killbill.github.io/slate/#invoice-adjust-an-invoice-item[adjust an invoice item]. An invoice item adjustment is often generated by an operator and is associated with a refund. In such a case, a new invoice item of type `ITEM_ADJ` is added to the invoice. If the invoice was already paid for, another invoice item of type `CBA_ADJ` is also added so as to adjust the credit amount in the next invoice.
+
+There are several cases in this scenario, let us understand each one.
+
+=== Case 1: Invoice item adjustment when invoice is not paid
+
+This scenario demonstrates what happens when an invoice item belonging to an unpaid invoice is https://killbill.github.io/slate/#invoice-adjust-an-invoice-item[adjusted].
+
+. Let us assume that an invoice is generated corresponding to a recurring subscription for `$100`. It will have an invoice item of type `RECURRING` and amount `100`. Assuming that the account does not have a valid default payment method, the invoice is not automatically paid, so its `invoice balance` is `100`.
+
+. When the https://killbill.github.io/slate/#invoice-adjust-an-invoice-item[invoice item is adjusted] (say for `$10`), the invoice is modified. A new invoice item is added of type `ITEM_ADJ` and amount `-10`. 
+
+So, `CHARGED AMOUNT` and `BALANCE` are calculated as follows:
+
+[source,java]
+----
+CHARGED AMOUNT = FIXED + RECURRING + EXTERNAL_CHARGE + USAGE + TAX + ITEM_ADJ + REPAIR_ADJ + CREDIT_ADJ
+               = 0 + 0 + 100 + 0 + 0 + (-10) + 0 + 0
+               = 90
+               
+INVOICE BALANCE = CHARGED AMOUNT + SUM(CBA_ADJ) - (AMOUNT PAID + AMOUNT REFUNDED)
+                = 90 + 0 - (0 + 0)  
+                = 90               
+---- 
+
+[[invoice_item_adjustment_for_paid_invoice]]
+=== Case 2: Invoice item adjustment when invoice is paid
+
+This scenario demonstrates what happens when an invoice item belonging to a paid invoice is https://killbill.github.io/slate/#invoice-adjust-an-invoice-item[adjusted]. 
+
+. Let us assume that an invoice is generated corresponding to a recurring subscription for `$100`. It will have an invoice item of type `RECURRING` and amount `100`. Assuming that the account has a valid default payment method, the invoice is automatically paid, so its `invoice balance` is `0`.
+
+. When this https://killbill.github.io/slate/#invoice-adjust-an-invoice-item[invoice item is adjusted], the invoice is modified. As before, an invoice item of type `ITEM_ADJ` and amount `-10` is added. In addition, another invoice item of type `CBA_ADJ` and amount `10` is added. Since the invoice is paid for, the `CBA_ADJ` item is added by the system so that the credit can be adjusted in future invoices.
```

Comment:
` In addition, another invoice item of type `CBA_ADJ` and amount `10` is added.` -> ` In addition, another invoice item of type `CBA_ADJ` and amount `10` is added to bring the balance to Zero. The addition of this positive `CBA_ADJ` item represents a credit generation by the system`

Reply:
Fixed